trivy-operator.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: trivy-operator
+  version: "0.25.0"
+  epoch: 8
+  description: "Kubernetes-native security toolkit that finds and reports vulnerabilities and misconfigurations"
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aquasecurity/trivy-operator
+      tag: v${{package.version}}
+      expected-commit: 0816542808e81dc318a34a1ce8a5cd004c59dfdd
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/containerd/containerd@v1.7.27
+        github.com/containerd/containerd/v2@v2.0.4
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        github.com/golang-jwt/jwt/v5@v5.2.2
+        golang.org/x/oauth2@v0.27.0
+        helm.sh/helm/v3@v3.17.3
+
+  - uses: go/build
+    with:
+      packages: "./cmd/trivy-operator"
+      output: trivy-operator
+      ldflags: |-
+        -X main.version=${{package.version}}
+        -X main.commit=$(git rev-parse HEAD)
+        -X main.date=$(date -u -d "@$SOURCE_DATE_EPOCH" +"%Y-%m-%dT%H:%M:%SZ")
+
+update:
+  enabled: true
+  github:
+    identifier: aquasecurity/trivy-operator
+    strip-prefix: v
+
+test:
+  environment:
+    environment:
+      OPERATOR_NAMESPACE: "trivy-system"
+      KUBERNETES_SERVICE_HOST: "127.0.0.1"
+      KUBERNETES_SERVICE_PORT: "32764"
```

Comment:
these two environment variables should not be required, `make docker-test/trivy-operator`